addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductsWidgetControlPanel.js

```diff
@@ -37,6 +38,37 @@ odoo.define('point_of_sale.ProductsWidgetControlPanel', function(require) {
             this.trigger('switch-category', 0);
             this.trigger('update-search', productName);
         }
+        async loadProductFromDB() {
+            if(!this.searchWordInput.el.value)
+                return;
+
+            try {
+                let ProductIds = await this.rpc({
+                    model: 'product.product',
+                    method: 'search',
+                    args: [[['name', 'ilike', this.searchWordInput.el.value + "%"]]],
+                    context: this.env.session.user_context,
+                });
+                if(!ProductIds.length) {
+                    this.showPopup('ErrorPopup', {
+                        title: this.env._t(''),
+                        body: this.env._t("No product found"),
+                    });
+                } else {
+                    await this.env.pos._addProducts(ProductIds);
+                }
+                this.trigger('update-product-list');
+            } catch (error) {
+                if (isRpcError(error) && error.message.code < 0) {
```

Comment:
I guess you will have to use the `identifyError` instead of this `isRpcError`. The `isRpcError` doens't work anymore since the merge of wowl.